index.js

```diff
@@ -22,7 +24,7 @@ module.exports = function makeIPFSFetch ({ ipfs }) {
     headers.Allow = SUPPORTED_METHODS.join(', ')
 
     async function getStat (path) {
-      return exporter(path, ipfs.block, { signal, preload: false })
+      return exporter(path, ipfs.block, { signal, preload: false, ipfsTimeout })
```

Comment:
```suggestion
      return exporter(path, ipfs.block, { signal, preload: false, timeout: ipfsTimeout })
```

Should all of these be changed to `timeout: ipfsTimeout`? I think that's what it says in the docs.

Also would you be able to add a unit test to make sure the timeout actually works?

Reply:
looks like i got used to copy and pasting lol, fixed now. i will go ahead. i will work on the unit tests now.

Reply:
made the test and tested on a CID that is offline and it timeouts like it should